layouts/partials/nav.html

```diff
@@ -1,52 +1,83 @@
-              <ul class="navbar-nav  ml-auto">
-      {{ range .Site.Menus.main }}
-        {{ if .HasChildren }}
-        <li class="nav-item dropdown">
-          <a href="#" class="nav-link dropdown-toggle"  id="navbarDropdownMenuLink" data-toggle="dropdown">
-            {{ .Pre }} <span>{{ .Name }}</span>
-            <span class="caret"></span>
-          </a>
-          <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdownMenuLink">
-            {{ range .Children }}
-            <a class="dropdown-item" href="{{ .URL | relLangURL }}"{{ if $.IsHome }} data-target="{{ .URL }}"{{ end }}>
+      <button class="navbar-toggler navbar-toggler" type="button" data-toggle="collapse" data-target="#navigation"
+        aria-controls="navigation-index" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-bar bar1"></span>
+        <span class="navbar-toggler-bar bar2"></span>
+        <span class="navbar-toggler-bar bar3"></span>
+      </button>
+    </div>
+    <div class="collapse navbar-collapse justify-content-end" id="navigation"
+      data-nav-image="{{ "img/blurred-image-1.jpg" | relURL }}">
+      <ul class="navbar-nav my-4 mr-auto">
+        {{ range .Site.Menus.main }}
+          {{ if .HasChildren }}
+          <li class="nav-item dropdown">
+            <a href="#" class="nav-link dropdown-toggle" id="navbarDropdownMenuLink" data-toggle="dropdown">
+              {{ .Pre }} <span>{{ .Name }}</span>
+              <span class="caret"></span>
+            </a>
+            <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdownMenuLink">
+              {{ range .Children }}
+              <a class="dropdown-item" href="{{ .URL | relLangURL }}" {{ if $.IsHome }} data-target="{{ .URL }}"
+                {{ end }}>
                 {{ .Pre }}
                 <span>{{ .Name }}</span>
               </a>
-            {{end}}
-          </div>
-        </li>
-        {{ else }}
-        <li class="nav-item">
-          <a class="nav-link" href="{{ .URL | relLangURL }}"{{ if $.IsHome }} data-target="{{ .URL }}"{{ end }}>
-            {{ .Pre }}
-            <span>{{ .Name }}</span>
-          </a>
+              {{end}}
+            </div>
+          </li>
+          {{ else }}
+          <li class="nav-item">
+            <a class="nav-link" href="{{ .URL | relLangURL }}" {{ if $.IsHome }} data-target="{{ .URL }}" {{ end }}>
+              {{ .Pre }}
+              <span>{{ .Name }}</span>
+            </a>
+          </li>
+          {{ end }}
+        {{ end }}
+      </ul>
+  <!-- Sidebar social network icons -->
+    <ul class="navbar-nav my-auto visible-xs align-items-center">
+      <li>
+        {{ with .Site.Params.mastodon }}<a target="_blank" href="{{ . }}" title="{{ . }}"><i class="fab fa-mastodon fa-2x mx-2"></i></a>{{ end }}
```

Comment:
```suggestion
        {{ with .Site.Params.mastodon }}<a target="_blank" href="{{ . }}" title="Mastodon"><i class="fab fa-mastodon fa-2x mx-2"></i></a>{{ end }}
```

Give it up for the blind folks!!

Reply:
Also makes the `{{ with }}` unnecessary and thus your code more readable.

Reply:
Ok!